letta/server/rest_api/routers/v1/tools.py

```diff
@@ -243,6 +243,7 @@ def add_composio_tool(
         raise HTTPException(
             status_code=400,  # Bad Request
             detail={
+                "error_code": "EnumStringNotFound",
```

Comment:
key name `code` instead of `error_code` to keep it consistent with the other callsites on front end and backend